test/test_nn.py

```diff
@@ -6236,22 +6260,34 @@ def multimarginloss_weights_no_reduce_test():
     ),
     dict(
         module_name='Upsample',
-        constructor_args=(12, None, 'linear'),
+        constructor_args=(12, None, 'linear', False),
         input_size=(1, 2, 4),
         desc='linear_1d',
     ),
     dict(
         module_name='Upsample',
-        constructor_args=((4, ), None, 'linear'),
+        constructor_args=((4, ), None, 'linear', False),
         input_size=(1, 2, 3),
         desc='linear_tuple_1d',
     ),
     dict(
         module_name='Upsample',
-        constructor_args=(None, 4, 'linear'),
+        constructor_args=(None, 4, 'linear', False),
         input_size=(1, 2, 4),
         desc='linear_scale_1d',
     ),
+    dict(
+        module_name='Upsample',
+        constructor_args=(12, None, 'linear', True),
+        input_size=(1, 2, 4),
+        desc='linear_1d_align_corners',
+    ),
```

Comment:
What about the `(4, ), None, 'linear'` test?

Reply:
`align_corners` has no impact on the `tuple vs scalar` input, so I figured we can skip that to save some CI time. But I can add it if you think it is necessary.

Reply:
No, I don't think it is necessary.